lib/pages/sign_up.dart

```diff
@@ -47,6 +47,11 @@ class _SignUpState extends State<SignUp> {
   void _sendLink(String email) async {
     if (_formKey.currentState!.validate()) {
       final stateRegistration = Get.find<RegistrationCubit>().state;
+      if (stateRegistration is RegistrationFailed) {
```

Comment:
Why nest them?
```dart
if (stateRegistration is RegistrationFailed && stateRegistration.emailExists) {
```
isn't this more ergonomic?

Reply:
Yeap